io/plugins/eu.esdihumboldt.hale.io.gml.test/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter2Test.groovy

```diff
@@ -381,4 +381,72 @@ class StreamGmlWriter2Test {
 
 		result
 	}
+
+	@Test
+	void testNumbers() throws Exception {
+		double doubleNumber = 123456789.123456789
+		double expected = java.lang.Double.parseDouble(doubleNumber.toString())
```

Comment:
Maybe add a comment here, that this value is then expected to use scientific notation?
Otherwise I think it will be hard for someone else to understand later on why this test is done that way.